### DISCUSSION TITLE
Record of substitutions in play-by-play

### DISCUSSION THREAD

**Discussion by JVAdams** [saiemgilani/wehoop#17] [Ideas]
Thanks for creating **wehoop**. It's great!

One thing I'd like to be able to do is calculate statistics for players based on only those times when they're in the game (and not on the bench). In order to do that, I'd need a record of substitutions. Any plans to include that information in the play-by-play data?

**Comment by saiemgilani** [comment]
Well if you can find it on ESPN, I'll include it. I'm not able to produce data that don't exist. Try wbigballR

**Reply by JVAdams** [reply]
Thanks. I'll give that a look.
